documentation/Demo-Server-Guide.md

```diff
@@ -96,15 +99,14 @@ _To see the actual Docker commands that are constructued, please inspect the fil
   - port 8000
   - WEB_HOST url
   - JWT_SECRET (this can be any random string, but should be a decent length, say > 24 alphanumeric characters. There's no need to record this key anywhere, as it can change anytime -- that just means existing JWTs become invalid, so users will need to re-login)
-    The system will be launched with “core_snapshot” data. Upload and run a new snapshot as required
+  
+  The system will be launched with “core_snapshot” data. Upload and load a new snapshot as required. 
 
-## JUMP to section for a basic server upgrade
-
-- [docker-compose](#docker-compose)
```

Comment:
Can you keep this please?

Reply:
It didn't make sense what it refers to, and what do we mean by "basic server upgrade"? 

I still have the link to the docker-compose section in the same place, but with a bit more clarification what it actually means:

>   Note that this `docker run` procedure resets the container every time it is restarted, so any data changes are wiped out. In order to avoid this, we need persist the data using Docker volumes. See the [docker-compose](#docker-compose) section below for information regarding this.